pkg/database/user.go

```diff
@@ -227,6 +229,21 @@ func (u *User) CreateFirebaseUser(ctx context.Context, fbAuth *auth.Client) (boo
 	return false, nil
 }
 
+// PasswordChanged updates the last password change timestamp of the user.
+func (db *Database) PasswordChanged(email string, t time.Time) error {
+	q := db.db.
+		Model(&User{}).
+		Where("email = ?", email).
+		Update("last_password_change", t.UTC())
```

Comment:
I think we should use UpdateColumn here instead (which will bypass validations and whatnot. Generally faster.